geps/gep-1713/index.md

```diff
@@ -1,7 +1,7 @@
 # GEP-1713: ListenerSets - Standard Mechanism to Merge Multiple Gateways
 
 * Issue: [#1713](/kubernetes-sigs/gateway-api/issues/1713)
-* Status: Provisional
+* Status: Experimental
```

Comment:
Did you update the `metadata.yaml` as well?

Reply:
yup - done in the earlier PR